packages/stem-begroot/src/step-2/selected-overview.tsx

```diff
@@ -44,9 +44,18 @@ export const BegrotenSelectedOverview = ({
         {selectedResources.map((resource) => (
           <div key={`budget-overview-row-${resource.id}`}>
             <div className="budget-two-text-row-spaced">
-              <Paragraph>{resource.title}</Paragraph>
+              <section className='budget-overview-row'>
+                <Image
+                  height={'4rem'}
+                  width={'4rem'}
+                  src={resource.images?.at(0)?.url || ''}
+                />
+                <Paragraph>{resource.title}</Paragraph>
+              </section>
               {typeIsBudgeting ? (
-                <Paragraph><Strong>{resource.budget}</Strong></Paragraph>
+                <Paragraph>
+                  <Strong>&euro;{resource.budget || 0}</Strong>
```

Comment:
Ik weet niet of het bij deze PR hoort, maar ik denk dat we alle bedragen ook even door een formatting moeten halen, dat je nette bedragen krijgt als `€ 34.995`